Library/Homebrew/utils/analytics.sh

```diff
@@ -1,30 +1,42 @@
 setup-analytics() {
-  # User UUID file. Used for Homebrew user counting. Can be deleted and
-  # recreated with no adverse effect (beyond our user counts being inflated).
+  HOMEBREW_GIT_CONFIG_FILE="$HOMEBREW_REPOSITORY/.git/config"
+
+  # User UUID file. This file has been deprecated in favor of
+  # homebrew.analyticsuuid in $HOMEBREW_REPOSITORY/.git/config. If detected,
+  # the UUID will be written to the aforementioned git config file, and the
+  # original UUID file will be removed.
   HOMEBREW_ANALYTICS_USER_UUID_FILE="$HOME/.homebrew_analytics_user_uuid"
```

Comment:
:+1:  Yep, that's the only file it appears in. Should `local`-ize it to minimize scope and avoid looking like an API element.

```
$ grep -rl HOMEBREW_ANALYTICS_USER_UUID_FILE Homebrew
Homebrew/utils/analytics.sh
[/usr/local/Library on ⇄ master]
$
```